### PR TITLE
Transparent icons

### PR DESCRIPTION
**Pull request by jsoref**
w/o these changes, a number of the icons look poor against a non-white background (especially the grey used in some rows/tables)

(I'm pulling in #2 because the work there was better)

### CODE REVIEW

**Comment by mpptx** [comment]
Hi there, it looks like those icons still do not have transparent background:
- https://github.com/jenkinsci/modernstatus-plugin/blob/master/src/main/webapp/48x48/disabled.png
- https://github.com/jenkinsci/modernstatus-plugin/blob/master/src/main/webapp/48x48/waiting.gif
Would you please fix them?

**Comment by jsoref** [comment]
@mpptx: This requires someone w/ permission to merge to merge.

Both of those are fixed in this PR:

```html
data:text/html,<body bgcolor=red><img src="https://raw.githubusercontent.com/jsoref/modernstatus-plugin/transparent/src/main/webapp/48x48/disabled.png"> <img src="https://raw.githubusercontent.com/jsoref/modernstatus-plugin/transparent/src/main/webapp/48x48/waiting.gif">
```

**Comment by mpptx** [comment]
Thanks a lot for your reply @jsoref, so we’re both waiting for this to be fixed…

--

Pozdrawiam / with regards

Michał



On 4 Feb 2021, at 18:55, Josh Soref <notifications@github.com<mailto:notifications@github.com>> wrote:



@mpptx<https://github.com/mpptx>: This requires someone w/ permission to merge to merge.

Both of those are fixed in this PR:

data:text/html,<body bgcolor=red>

**Comment by mpptx** [comment]
@daniel-beck @sschuberth could you please take a look at this? I think it's not a big deal to have those changes merged and it would make at least two people happier :)

**Comment by sschuberth** [comment]
@mpptx I don't have the rights to approve PRs in this repo...

**Comment by timja** [comment]
@olivervinn are you still maintaining this? / would you like someone else to take over?

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
------- | ----
-Build Now | ![Build Now](https://raw.githubusercontent.com/ovinn/modernstatus-plugin/master/src/main/webapp/24x24/clock.png)
-Delete Project | ![Delete Project](https://raw.githubusercontent.com/ovinn/modernstatus-plugin/master/src/main/webapp/24x24/edit-delete.png)
-Workspace | ![Workspace](https://raw.githubusercontent.com/ovinn/modernstatus-plugin/master/src/main/webapp/24x24/folder.png)
+# Modern Status for Jenkins [![Build Status](https://ci.jenkins.io/job/plugins/job/modernstatus-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/plugins/job/modernstatus-plugin/job/master/)
+
+Providing a modern look to Jenkins Status indicators. 
+
+https://wiki.jenkins-ci.org/display/JENKINS/Modern+Status+Plugin
+
```
Comment (timja) [inline README.md]:
```suggestion
```

**Review by timja — APPROVED** [review]

**Pull request merged**